redmine_to_kanboard.rb

```diff
@@ -77,7 +77,7 @@ def color_id
   end
 
   def task_title
-    @redmine_issue.subject
+    "Redmine \##{@redmine_issue.id}: #{@redmine_issue.subject}"
```

Comment:
it seems quite long and the information is in the link already, what's the usecase? would it be better to have shorten it to "RM #123"? I can imagine a greasemonkey script that would simplify navigation :-)

Reply:
What about just `#1234` instead of `Redmine #1234` or `RM #1234`?